### PR TITLE
User/le2tang/elevator ctrl

### PR DESCRIPTION
**Pull request by le2tang**

### CODE REVIEW

File: nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py [hunk 1]
```diff
@@ -100,16 +100,21 @@ def get_items_phase(self):
         # Arrived at stock area
```
Comment (FelixKHWong) [inline nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py:100]: should these changes be part of this PR? I feel like these should go with manipulation changes

File: nightingale_manipulation/src/nightingale_manipulation/manipulation_action_client.py [hunk 2]
```diff
@@ -190,51 +578,99 @@ def __init__(self):
...
+    manipulation.jnt_ctrl.cmd_torso(manipulation.jnt_ctrl.torso_home_joint_values)
+    manipulation.jnt_ctrl.cmd_head(manipulation.jnt_ctrl.head_home_joint_values)
+
+    # close the gripper
+    manipulation.gpr_ctrl.close_right()
+
+    # Move left arm to home with joint ctrl
+    manipulation.jnt_ctrl.cmd_left_arm(manipulation.jnt_ctrl.left_arm_home_joint_values)
+
+    manipulation.jnt_ctrl.cmd_torso([0.3])
```
Comment (FelixKHWong) [inline nightingale_manipulation/src/nightingale_manipulation/manipulation_action_client.py:641]: how did you determine this value for the torso and what significance does it have?

**Comment by younesr1** [comment]
this PR shows a diff for many things that i dont think are related to torso and head control. its a little hard to parse through. could u fix that please?
